.circleci/config.yml

```diff
@@ -173,6 +173,27 @@ jobs:
             - dist
             - node_modules
 
+  prepare-integreat-delivery: # Move release notes, bump version and tag commit
+    docker:
+      - image: circleci/node:12.16.3
+    resource_class: small
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Calculate next version
+          command: echo "export NEW_VERSION_NAME=$(build/next-version calc | jq .versionName)" >> ${BASH_ENV}
+      - run:
+          name: Create jira release
+          command: build/create-jira-release createRelease ${NEW_VERSION_NAME} --private-key ${JIRA_PRIVATE_KEY} --access-token ${JIRA_OAUTH_TOKEN} --consumer-key ${JIRA_CONSUMER_KEY}
```

Comment:
To be consistent this should be `create_release` as `move_to` and `bump_to`

Reply:
Yeah, I wanted to ask this anyway. We are mixing `-` and `_` all the time. Is there a convention for scripts? Or should we just decide on either one? I am fine with both @maxammann